rclcpp/include/rclcpp/node_interfaces/node_base.hpp

```diff
@@ -114,13 +114,9 @@ class NodeBase : public NodeBaseInterface
   get_associated_with_executor_atomic() override;
 
   RCLCPP_PUBLIC
-  rcl_guard_condition_t *
+  rclcpp::GuardCondition &
   get_notify_guard_condition() override;
```

Comment:
This API also should be mentioned in the notice we put in the Humble release page. It is even less likely to be used outside of the core, but still it should be mentioned.

Reply:
Done in https://github.com/ros2/ros2_documentation/pull/2086